tmpfiles.d/avahi-service-writers.conf

```diff
@@ -0,0 +1,3 @@
+# Type Path    Mode UID  GID  Age Argument
+d /etc/avahi/services 0755 root avahi-service-writers
+Z /etc/avahi/services 0755 root avahi-service-writers
```

Comment:
Is the `Z` line necessary? The documentation for `d` (in `man tmpfiles.d`) says “The mode and ownership will be adjusted if specified and the directory already exists.”.

Reply:
So `Z` will `Recursively set the access mode, group and user, and restore the SELinux security context of a file or directory if it exists`. My thinking here was that if someone had already touched that file then the `Z` will at least reset the permissions so that it is writable.

Then again, I don't see that happening on a normal system so I'm happy to drop this line. Drop it?

Reply:
I don't see it as being harmful, but you're right -- the directory permissions should be enough, so let's remove it for now.